File: conductor-modules/support/src/test/java/com/bluelinelabs/conductor/support/util/TestActivity.java
package com.bluelinelabs.conductor.support.util;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.bluelinelabs.conductor.support.R;

public class TestActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTheme(R.style.Theme_AppCompat);
    }
}

File: conductor-modules/viewpager2/src/test/java/com/bluelinelabs/conductor/viewpager2/util/TestActivity.kt
package com.bluelinelabs.conductor.viewpager2.util

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.bluelinelabs.conductor.R

class TestActivity:AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setTheme(R.style.Theme_AppCompat)
    }
}